modules/@apostrophecms/doc-type/ui/apos/components/AposDocMoreMenu.vue

```diff
@@ -5,6 +5,8 @@
     :disabled="disabled"
     menu-placement="bottom-end"
     @item-clicked="menuHandler"
+    @open="$emit('menuOpen')"
+    @close="$emit('menuClose')"
```

Comment:
let our UI know if the menu is left open so we can do things like keep it visible